recipes/python/config/access_hosts_api.py

```diff
@@ -0,0 +1,24 @@
+import requests
+
+vmware_access_hosts_url = "/config/vmware/access-hosts/"
+content_type_header = "application/vnd.netbackup+json;version=3.0"
+accept_header = "application/vnd.netbackup+json;version=3.0"
+
+def get_access_hosts(base_url, jwt, filter):
+    headers = {'Accept': accept_header, 'Authorization': jwt}
+    long_url = base_url + vmware_access_hosts_url + filter
+    response = requests.get(long_url, headers=headers, verify=False)
+    return response
+
+def add_access_host(base_url, jwt, hostName):
+    headers = {'Content-Type': content_type_header, 'Authorization': jwt}
+    long_url = base_url + vmware_access_hosts_url
+    data = {'data':{'type':'accessHostRequest', 'id':'vmware', 'attributes':{'hostname':hostName, 'validate': 'false'}}}
```

Comment:
Is the `validate` field really supposed to have the _string_ value `"false"` and not the Boolean value `False`?